grip/command.py

```diff
@@ -23,6 +23,7 @@
   --clear           Clears the cached styles and assets and exits
   --export          Exports to <path>.html or README.md instead of serving,
                     optionally using [<address>] as the out file (- for stdout)
+  -b --browser         Option to not open a tab in the browser.
```

Comment:
Small tweak. Remove the word 'not'. I can do this after pulling it in if you want.